locales/templates/app.pot

```diff
@@ -332,10 +330,10 @@ msgctxt "first_run_page1_title"
 msgid "Thanks for choosing %1$s"
 msgstr ""
 
+#. message for first page of FirstRun, Argument 1 will be "Mozilla"
```

Comment:
Nit: should start with a capital letter. Also probably better to say "%1$s" is Mozilla since there's just one argument? or at least calling out that the argument is %1$s